astrocut/cube_cut.py

```diff
@@ -40,51 +41,51 @@ def __init__(self):
         Initiazation function.
         """
 
-        self.cube_wcs = None  # WCS information from the image cube
-        self.cutout_wcs = None  # WCS information (linear) for the cutout
+        self.cube_wcs = None # WCS information from the image cube
+        self.cutout_wcs = None # WCS information (linear) for the cutout
         self.cutout_wcs_fit = {'WCS_MSEP': [None, "[deg] Max offset between cutout WCS and FFI WCS"],
                                'WCS_SIG': [None, "[deg] Error measurement of cutout WCS fit"]}
         
-        self.cutout_lims = np.zeros((2, 2), dtype=int)  # Cutout pixel limits, [[ymin,ymax],[xmin,xmax]]
-        self.center_coord = None  # Central skycoord
+        self.cutout_lims = np.zeros((2, 2), dtype=int) # Cutout pixel limits, [[ymin,ymax],[xmin,xmax]]
+        self.center_coord = None # Central skycoord
         
-        # Extra keywords from the FFI image headers (TESS specific)
+        # Extra keywords from the FFI image headers in SPOC
         self.img_kwds = {"BACKAPP": [None, "background is subtracted"],
```

Comment:
Would it make sense to read in keywords from a config file? Would it make future maintenance easier?

Reply:
Hi Jeff, yes I agree I think moving things into a config file would help with the troubleshooting. I've created Issue # https://github.com/spacetelescope/astrocut/issues/66 for this so I can tackle it after the initial TICA deployment.